src/Data/Bifunctor/Clown.hs

```diff
@@ -23,47 +23,58 @@ import Data.Coerce
 import Data.Biapplicative
 import Data.Bifoldable
 import Data.Bifunctor
+import Data.Bifunctor.ShowRead
 import Data.Bifunctor.Unsafe
 import Data.Bitraversable
 import Data.Functor.Contravariant
 import Data.Data
 import Data.Functor.Classes
 import GHC.Generics
+import Text.Read (Read (..), readListPrecDefault)
 
 -- | Make a 'Functor' over the first argument of a 'Bifunctor'.
 --
 -- Mnemonic: C__l__owns to the __l__eft (parameter of the Bifunctor),
 --           joke__r__s to the __r__ight.
 newtype Clown f a b = Clown { runClown :: f a }
-  deriving (Eq, Ord, Show, Read, Data, Generic, Generic1)
+  deriving (Eq, Ord, Data, Generic, Generic1)
 
-instance (Eq1 f, Eq a) => Eq1 (Clown f a) where
-  liftEq = liftEq2 (==)
+instance Eq (f a) => Eq1 (Clown f a) where
```

Comment:
This `Eq1` instance for `Clown` is a bit interesting in that it has an `Eq (f a)` context rather than an `Eq1 f` one. Moreover, `Joker`'s `Eq1` instance differs in that its context _does_ have an `Eq1 f` context. Is the difference that in the case of `Joker`, the field mentions the last type parameter, whereas in the case of `Clown` the field doesn't mention the last type parameter, which makes us default to using `Eq` instead?

Reply:
Yes, indeed. This is the sort of thing that makes people complain about `fmap` only operating on the second component of a pair; it's symmetrical math that goes asymmetrical in Haskell.